tests/unittests/BackendTestUtils.cpp

```diff
@@ -554,7 +585,7 @@ void inferBasicConvNet(Tensor *inputs, Tensor *out, BackendKind kind,
   Function *F = mod.createFunction("main");
   auto *var = VarFrom(inputs);
   auto *tr = F->createTranspose("tr", var, {0, 2, 3, 1});
-  auto *conv = F->createConv("conv", tr, convDepth, 5, 2, 1, 1);
+  auto *conv = F->createConv("conv", tr, convDepth, 5, 2, {1, 1, 1, 1}, 1);
```

Comment:
do you check in any of the tests all overloads of create conv (including the one which takes single pad).

Reply:
Most of the createConv in previous tests are not changed, which checks the single pad. This change and the test "TEST_P(CPUOnly, nonSquarePaddingConvTest)" in this PR checks the two createConv with "pads". So all 4 createConv functions are tested.